### DISCUSSION TITLE
!! Not a bug, just an Idea (unrelated to Firefox) !!

### DISCUSSION THREAD

**Discussion by scientiac** [rafaelmardojai/firefox-gnome-theme#426] [Ideas]
First of all You are great!!! I can see that you're very talented and wow my librewolf looks just like the gnome-web!.

I just wanted to ask if you could make one for Spotify - using spicetify - from what I can understand it's using CSS as well??

Sorry if this is not the right place for this request. It's fine you delete it without replying too.

**Comment by rafaelmardojai** [comment]
I use Spotify, so If I find some free time I can try something. 

The only problem si that I use Fedora Silverblue and installing Spicetify there seems complicated.
